### DISCUSSION TITLE
are other networks supported?

### DISCUSSION THREAD

**Discussion by meapulpa** [apohl79/audiogridder#1408] [Q&A]
Hello,
I'd like to know if it's possible to use dedicated digital audio networks to transfer audio/video from audiogridder. (Dante, AES67, Madi...) Some very interesting routing possibilities would be made possible and network latence/stability/quality would surely be optimal. 
I would also like to know if there is a clock signal that is transferred over the network.

Anyway, thanks for the amazing work and good luck.

**Comment by vincentark** [comment]
For sure no clock, it is buffers, so grouped packets of audio data, not an audio stream.
I have no idea about your first question, I think not cause it would possibly Interrupt the Dante etc stream

**Comment by DeadlyMixStudios** [comment]
Should be fine. Waves Soundgrid runs through a Dante switch without issue and the person I heard of AG from does it and recommended it to me as a replacement for our WSG server because Soundgrid requires this awful WSG-Dante Bridge that has to be used as an interface (which is why its awful), cutting our channel count from 512x512 down to 64x64. This is for the Dolby Atmos mastering room, so we need at least 300x300, maybe more. Point being that its been tried and people who've done it recommend it. So, you should be fine. Our hybrid console runs on our Dante network. We have a 3rd NIC hardwired in so that we have 1 for the Dante PCIe, the added NIC for DVS (system audio), and the other native NIC for internet. The internet is all we separate.
